tests/test_qaneeded.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from bugbug import bugzilla
+from bugbug.models.qaneeded import QANeededModel
+
+
+def test_get_qaneeded_labels():
+    model = QANeededModel()
+    classes = model.get_labels()
+    assert not classes[1389220]
+    assert classes[1389223], 'Bug should contain qawanted in a field'
+    assert classes[1390433], 'Bug should contain qe-verify in a field'
+
+
+def test_rollback():
+    model = QANeededModel()
+
+    histories = {}
+    for bug in bugzilla.get_bugs():
+        histories[int(bug['id'])] = bug
+
+    def contains_field(bug_id):
```

Comment:
This is redundant with the other function, you can remove it and just always use the other function.